--- src/Lawn/Widget/SeedChooserScreen.cpp
+++ src/Lawn/Widget/SeedChooserScreen.cpp
@@ -311,6 +311,18 @@
 	return mApp->mGameMode != GAMEMODE_CHALLENGE_BIG_TIME && mApp->mGameMode != GAMEMODE_CHALLENGE_BUNGEE_BLITZ && mApp->mGameMode != GAMEMODE_CHALLENGE_ART_CHALLENGE_1 && mApp->mGameMode != GAMEMODE_CHALLENGE_ART_CHALLENGE_2;
 }
 
+int SeedChooserScreen::GetAlmanacIndex(SeedType theSeedType)
+{
+	int maxPlants = gModRegistry.GetTotalAlmanacPlants();
+	for (int i = 0; i < maxPlants; i++)
+	{
+		if ((SeedType)gModRegistry.GetAlmanacPlantAt(i) == theSeedType)
+			return i;
+	}
+	return 0;
+}
+
 void SeedChooserScreen::GetSeedPositionInChooser(int theIndex, int& x, int& y)
